### PR TITLE
parenthesis (v5)

### PR DESCRIPTION
**Pull request by rvilarl**
Minimal visual differences:

current
EasyScore Draw_Parenthesised Bravura svg_current
reference
EasyScore Draw_Parenthesised Bravura svg_reference

### CODE REVIEW

File: src/parenthesis.ts [hunk 1]
```diff
@@ -67,19 +63,22 @@ export class Parenthesis extends Modifier {
     super();
 
     this.position = position ?? Modifier.Position.LEFT;
+    if (this.position == Modifier.Position.RIGHT) {
+      this.text = '\uE0F6'; /*noteheadParenthesisRight*/
+    } else if (this.position == Modifier.Position.LEFT) {
+      this.text = '\uE0F5'; /*noteheadParenthesisLeft*/
+    }
```
Comment (ronyeh) [inline src/parenthesis.ts:70]: Let's use === by default.

File: src/parenthesis.ts [hunk 2]
```diff
@@ -67,19 +63,22 @@ export class Parenthesis extends Modifier {
...
 
-    this.point = Tables.currentMusicFont().lookupMetric('parenthesis.default.point') ?? Note.getPoint('default');
-    this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.default.width'));
+    this.setFontSize(Note.getPoint('default'));
```
Comment (ronyeh) [inline src/parenthesis.ts:72]: This can be a different PR, but I think Note.getPoint() could have a clearer method name. Does it relate to font size, or to a point in 2D space?

File: src/parenthesis.ts [hunk 3]
```diff
@@ -67,19 +63,22 @@ export class Parenthesis extends Modifier {
...
   setNote(note: Note): this {
     this.note = note;
-    this.point = Tables.currentMusicFont().lookupMetric('parenthesis.default.point') ?? Note.getPoint('default');
-    this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.default.width'));
     if (isGraceNote(note)) {
-      this.point = Tables.currentMusicFont().lookupMetric('parenthesis.gracenote.point') ?? Note.getPoint('gracenote');
-      this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.gracenote.width'));
+      this.setFontSize(Note.getPoint('gracenote'));
+    } else {
+      this.setFontSize(Note.getPoint('default'));
```
Comment (ronyeh) [inline src/parenthesis.ts:81]: Why isn't it named Note.getFontSize()?

**Review by ronyeh — CHANGES_REQUESTED** [review]
Just some questions, but otherwise looks OK.

File: src/parenthesis.ts (hunk 1 above)
Comment (rvilarl) [reply src/parenthesis.ts:70]: :) done

File: src/parenthesis.ts (hunk 3 above)
Comment (rvilarl) [reply src/parenthesis.ts:81]: Very good idea! rather get the font from the note: note.getFont()

File: src/parenthesis.ts (hunk 2 above)
Comment (rvilarl) [reply src/parenthesis.ts]: No longer used.

File: src/parenthesis.ts (hunk 3 above)
Comment (rvilarl) [reply src/parenthesis.ts]: And remove note.getPoint

**Comment by ronyeh** [comment]
Thanks!

I'll get back to reviewing sometime in the middle of next week. Have a nice weekend (and take a deserved break, haha)!
